template/package.json

```diff
@@ -13,8 +13,11 @@
     {{#e2e}}
     "e2e": "node test/e2e/runner.js",
     {{/e2e}}
-    "test": "{{#unit}}npm run unit{{/unit}}{{#unit}}{{#e2e}} && {{/e2e}}{{/unit}}{{#e2e}}npm run e2e{{/e2e}}"{{#lint}},
-    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"{{/lint}}
+    "lint": "{{#eslint}}npm run lint:eslint{{/eslint}}{{#eslint}}{{#stylelint}} && {{/eslint}}{{/stylelint}}{{#stylelint}}npm run lint:stylelint{{/stylelint}}",
+    "test": "{{#unit}}npm run unit{{/unit}}{{#unit}}{{#e2e}} && {{/e2e}}{{/unit}}{{#e2e}}npm run e2e{{/e2e}}"{{#eslint}},
+    {{#eslint}}
+    "lint:eslint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"{{/eslint}}{{#stylelint}},
+    "lint:stylelint": "stylelint 'src/**/*.@(?(s)?(a|c)ss|vue|html)' --config ./.stylelintrc.js --extract"{{/stylelint}}
```

Comment:
Good question we can probably have sub conditions for syntax. And as using plugin we could do less. 

Scss is supported in node blob for files. 

Less is currently experimental, to i have avoided it currently. Also I think Less is very little used these days. Especially if starting a new project, as scaffolding is for new projects I think we can leave Less behind until stylelint fully supports it.